src/Shared/StackExchangeClientConnection.cs

```diff
@@ -188,9 +188,9 @@ internal SessionStateItemCollection DeserializeSessionStateItemCollection(RedisR
         {
             try
             {
-                MemoryStream ms = new MemoryStream((byte[])serializedSessionStateItemCollection);
-                BinaryReader reader = new BinaryReader(ms);
-                return SessionStateItemCollection.Deserialize(reader);
+                var bytes = (byte[])serializedSessionStateItemCollection;
```

Comment:
These "SessionState"-specific methods also don't belong in this "Shared" file. They should be moved into a file under `/RedisSessionStateProvider`

Reply:
Done